source/llvm/constants.d

```diff
@@ -654,6 +743,10 @@ static if (LLVM_Version >= asVersion(6, 0, 0))
 		                           LLVMDIFlagMultipleInheritance |
 		                           LLVMDIFlagVirtualInheritance
 	}
+} 
+
+static if (LLVM_Version >= asVersion(6, 0, 0)) 
```

Comment:
Why this change, are you saying `LLVMDWARFSourceLanguage`, which is already in llvm-d was wrongly supported for LLVM < 6?

Reply:
`LLVMDWARFSourceLanguage` was not changed, before it was in a big `static if (LLVM_Version> = asVersion (6, 0, 0))` block. Now I have split it.

Reply:
Ah, didn't catch that, thanks